x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/actions/hipchat/HipChatActionFactory.java

```diff
@@ -27,6 +29,7 @@ public HipChatActionFactory(TextTemplateEngine templateEngine, HipChatService hi
 
     @Override
     public ExecutableHipChatAction parseExecutable(String watchId, String actionId, XContentParser parser) throws IOException {
+        deprecationLogger.deprecated("hipchat actions are deprecated and will be removed in 7.0");
```

Comment:
I think this will trigger for every watch execution. If so, should we use `DeprecationLogger#deprecatedAndMaybeLog` instead to avoid spamming the deprecation logs.

Reply:
Good call, I switched to that.